app/views/bots/index.html.erb

```diff
@@ -4,6 +4,7 @@
       <div class="bot-dash-module-container col-md-4">
         <div class="bot-dash-module col-md">
           <h3><%= bot.name %></h3>
+          <p>by <%= user_link(User.with_role(:owner, bot).first) %></p>
```

Comment:
Could have `class="text-muted"` on this to look a bit more fancy. I'd probably put it in a `<small>` within the `<h2>`, as well, but that's just a stylistic choice, so I'm fine with whatever you think looks better.